src/Sulu/Bundle/ContentBundle/.gitignore

```diff
@@ -3,6 +3,7 @@
 /composer.phar
 /composer.lock
 /.sass-cache/
+/Resources/app/cache
```

Comment:
Not sure I understand - we fallback to the root `.gitignore` in this case, which should ignore any test files with a pattern, e.g. `/src/Sulu/Bundles/*/node_modules` f.e. but its another issue.